<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_dialog</class>
 <widget class="QDialog" name="login_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>373</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Aurora</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>221</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_access">
    <item>
     <widget class="QTextEdit" name="text_name">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="placeholderText">
       <string>Nome completo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="text_email">
      <property name="placeholderText">
       <string>E-mail</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="text_cpf">
      <property name="placeholderText">
       <string>CPF</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_register">
      <property name="text">
       <string>Registrar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/square logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Aurora - Vers√£o 0.20</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
